content/manuals/build-cloud/_index.md

```diff
@@ -65,6 +65,6 @@ Once you've signed up and created a builder, continue by
 [setting up the builder in your local environment](./setup.md).
```

Comment:
Can't leave a comment on the right line but first paragraph of this section still reads "and sign up for the starter plan" but there is no more "starter plan" or plan selection in DBC dashboard for users on Consolidated Plans.

Instead the user just goes through the "intro slideshow" and DBC is activated in the background unless they are a Free Personal account in which case they are given the option to start a 7 day trial.

cc @colinhemmings need exact wording